bounce.go

```diff
@@ -29,11 +30,16 @@ type DeliveryStats struct {
 	Bounces []BounceType
 }
 
-// GetDeliveryStats returns delivery stats for the server
+// GetDeliveryStats calls GetDeliveryStatsWithContext with empty context
 func (client *Client) GetDeliveryStats() (DeliveryStats, error) {
+	return client.GetDeliveryStatsWithContext(context.TODO())
```

Comment:
We can use `context.Background()` which is what requests with context end up using: https://golang.org/src/net/http/request.go?s=26997:27066#L805